### ISSUE TITLE
Should assemblies be removed if core gene alignment shows redundancies? 

### ISSUE THREAD

**Issue by chahatupreti** [sanger-pathogens/Roary#580]
I have a set of 100 bacterial genomes. I annotated them with Prokka and performed pangenome analysis with Roary. Roary outputs the core gene alignment file which I then used to generate a phylogenetic tree using RaxML. While running RaxML, the console output said -

`IMPORTANT WARNING - Found 13 sequences that are exactly identical to other sequences in the alignment. Normally they should be excluded from the analysis.`

My question is, should I remove these 13 sequences from my subsequent analyses based on this information? I first thought that this it would be obvious to remove these redundant/clonal sequences so that they don't mess up the statistics for gene enrichment etc. But a counterargument is that these 13 sequences are being called as exactly identical to other sequences in my database based on the **core gene alignment**. What about any differences these 13 assemblies may have (from the sequences these are supposedly identical to) in the non-core genome?

In other words, what if these sequences are actually completely unique but their uniqueness lies in terms of those genes that are not core genes, but those that are present in a subset of the assemblies?

Any insights would be great appreciated. I apologize for this not being a question directly about running Roary but about analysis based on its output.

**Comment by andrewjpage** [comment]
It depends, if you are analysing an outbreak and expect to have identical samples (or you've done replicates) then I'd probably leave them in. If these aren't from an outbreak, then something else might be going on. For example, maybe your core genome is tiny because theres a rogue sample of a different species in there.
